pkgs/top-level/all-packages.nix

```diff
@@ -19272,7 +19271,9 @@ in
 
   sysklogd = callPackage ../os-specific/linux/sysklogd { };
 
-  syslinux = callPackage ../os-specific/linux/syslinux { };
+  syslinux = callPackage ../os-specific/linux/syslinux {
+    stdenv = gcc9Stdenv; # Fails with "undefined reference to `strlen'" on gcc10
```

Comment:
OpenSUSE seems to have a patch for this one: https://build.opensuse.org/package/view_file/openSUSE:Factory/syslinux/syslinux-4.04-gcc10.diff?expand=1

Reply:
Thanks!